mkosi/__init__.py

```diff
@@ -1611,7 +1611,9 @@ def mount_api_vfs(root: Path) -> Iterator[None]:
 
 @contextlib.contextmanager
 def mount_cache(state: MkosiState) -> Iterator[None]:
-    if state.config.distribution in (Distribution.fedora, Distribution.mageia, Distribution.openmandriva):
+    if state.installer is not None:
```

Comment:
Just `state.installer` is good enough. This is a normal object which will never be false, so checking for None is just unnecessary pedantry.

(Also below.)